src/window-rules.c

```diff
@@ -73,6 +73,16 @@ view_matches_criteria(struct window_rule *rule, struct view *view)
 	}
 }
 
+void
+init_window_rule(struct window_rule *rule)
+{
+	if (!rule) {
+		return;
+	}
+	rule->has_custom_border = false;
+	memset(rule->custom_border_color, 0, sizeof(float)*4);
+}
+
```

Comment:
This function is not necessary because structs will be zero-initialized. If it were necessary, it shouldn't should exist only to set the color.